code/modules/shuttle/arrivals.dm

```diff
@@ -7,6 +7,7 @@
 	height = 15
 	dir = WEST
 	port_angle = 180
+	preferred_direction = WEST

```

Comment:
Also, if you scroll down to `Initialize()`. You'll notice that preferred_direction is set to dir. This is to make it so that the shuttle never has to rotate.